From: Sorah Fukumori <sorah@cookpad.com>
Date: Fri, 31 May 2019 03:12:27 +0900
Subject: Fix setup command test for bundler with program_suffix

This test fails with ruby built with --program-suffix.

This is an additional fix for #2766,
commit ec3c64a3de4048bec588896819df647c616cc6a8
---
 test/rubygems/test_gem_commands_setup_command.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/rubygems/test_gem_commands_setup_command.rb b/test/rubygems/test_gem_commands_setup_command.rb
index 3f9887e..4b16fc5 100644
--- a/test/rubygems/test_gem_commands_setup_command.rb
+++ b/test/rubygems/test_gem_commands_setup_command.rb
@@ -189,7 +189,7 @@ class TestGemCommandsSetupCommand < Gem::TestCase
         assert_path_exists File.join(bin_dir, "#{e}.bat")
       end
 
-      assert_path_exists File.join bin_dir, e
+      assert_path_exists File.join bin_dir, Gem.default_exec_format % e
     end
 
     default_dir = Gem::Specification.default_specifications_dir
